tests/common/src/tools/timer_test.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+
+#include "aos/common/tools/timer.hpp"
+
+using namespace aos;
+
+TEST(common, TimerCreateAndStop)
+{
+
+    auto       g_interrupted = 0;
```

Comment:
it is not gloabal variable. Just `interrupted`.

Reply:
Done